app/views/users/edit.html.erb

```diff
@@ -0,0 +1,57 @@
+<h1>USER Sign-up form:</h1>
+
+
+<!-- <form action="/users" method="post" accept-charset="UTF-8">
```

Comment:
Use the **_users\_path_** helper instead of "/users"

Reply:
Not need when creating pure html for as per the instructios